docs/doc/60-contributing/03-rfcs/20221017-multiple-catalog.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Multiple Catalog
+description: multiple catalog support for databend
+---
+
+- RFC PR: [datafuselabs/databend#0000](https://github.com/datafuselabs/databend/pull/0000)
+- Tracking Issue: [datafuselabs/databend#0000](https://github.com/datafuselabs/databend/issues/0000)
+
+## Summary
+
+Allow users to maintain multiple catalogs for the databend.
+
```

Comment:
I do not quite get what a `catalog` actually is from this RFC:

What's the definition of a `catalog`?
I.e., what's the essential data a `catalog` has, and how does a `catalog` access this data?

And what's the API trait a `catalog` provides?

Reply:
Fixed in https://github.com/datafuselabs/databend/pull/8255/commits/944f9aac065505146dbe9b297da41abdde0ed735